components/Guide/Guide.js

```diff
@@ -80,6 +80,15 @@ class Guide extends Component {
     );
   }
 
+  get contentComponent() {
+    const { content } = this.props;
+    if (typeof content === 'object') {
```

Comment:
Can we use [React.isValidElement](https://facebook.github.io/react/docs/top-level-api.html#react.isvalidelement) to verify it?